src/components/Header.tsx

```diff
@@ -106,7 +108,13 @@ const HeaderLinks = () => {
 const Header = (props: BoxProps) => {
```

Comment:
Do we want to fix the "padding" problem when resizing the viewport, or do we want to leave that for the full implementation? Or are we happy to merge this and work on the padding problem later?

I think the current site has a similar problem?

Reply:
Not really sure what is this problem about.

Reply:
I've just used the browser tools to measure the header total size:

| Simulated device | view port width |  view port height | `<header>` height |
| --- | --- | --- | --- |
| iPhone | 375 | 812 | 88 |
| iPad | 768 | 1024 | 83.2 |
| Desktop | 1440 | 900 | 98 |

This is really subtle, but I do notice the items below the header changing position when you resize the view port.

Reply:
You mean the filters switching from vertical stack to horizontal stack?